### PR TITLE
VideoPress: Pull and show title and description of VideoPress block

### LINKED ISSUE #26171: VideoPress: Add details video control
Create a new section called “Details” on the sidebar, which would contain a field labeled “Title” and a textearea labeled “Description”.
* If the video doesn’t have an explicitly added title yet, I’d display the file name as placeholder.

_Note: The title that would appear when pausing the player would be the explicitly added one. I would avoid showing the raw file name in the player._

Image

### LINKED ISSUE #26191: VideoPress: pull and render title and description
Get the `title` and `description` of the video hitting the media endpoint. Render these values in the DetailsControl component

### PR DESCRIPTION
**Pull request by retrofox**
In this PR:

* creates a hook to pull the video item data
* sanitize the data (decodeEntities)
* render the title and description in the DetailsControl when it's ready

Fixes https://github.com/Automattic/jetpack/issues/26191

#### Changes proposed in this Pull Request:

* Jetpack: pull and show title and description in the VideoPress Video sidebar

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Testing instructions:

* Upload a video (VideoPress)
* Edit `title` and `description` from the media gallery (wp-admin) or using Calypso
* Once the video data saves, do a hard refresh in the block editor
* Confirm the title and description have the proper data

wp-admin | Calypso | Block Editor
-----|----|----
Screen Shot 2022-09-13 at 13 34 31 | Screen Shot 2022-09-13 at 13 34 55 | Screen Shot 2022-09-13 at 13 35 19

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/utils/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const elementInstance = ( () => {
+	if ( document.implementation && document.implementation.createHTMLDocument ) {
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/utils/index.js]:
```suggestion
	if ( document.implementation?.createHTMLDocument ) {
```

File: projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/hooks/use-video-item/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const MEDIA_ENDPOINT = 'https://public-api.wordpress.com/rest/v1.1/videos/';
+
+export default function useVideoItem( guid ) {
+	const [ item, setItem ] = useState( {} );
+	const [ loading, setLoading ] = useState( false );
+
+	useEffect( () => {
+		async function fetchVideoItem() {
+			try {
+				setLoading( false );
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/hooks/use-video-item/index.js]: Why is it setting to `false`?

Comment (retrofox) [reply projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/hooks/use-video-item/index.js]: I need to move it down :-)

**Comment by brbrr** [comment]
Hey @retrofox. Just FYI, Jeremy is on sabbatical, so he won't be able to review your PRs for quite some time :)

**Review by oskosk — APPROVED** [review]
LGTM!

**Comment by retrofox** [comment]
r252570-wpcom

**Pull request merged**